android/build.gradle

```diff
@@ -63,8 +64,9 @@ subprojects {
         }
         kotlin {
             target "**/*.kt"
-            ktlint(deps.versions.ktlint).userData(['indent_size': '2', 'continuation_indent_size' : '2'])
-            ktfmt(deps.versions.ktfmt)
+            ktlint(deps.versions.ktlint).userData(['indent_size': '2', 'continuation_indent_size' : '4'])
+            // TODO: enable when ktfmt depends on google-java-format < v1.8
```

Comment:
@cgrushko We'll enable KTFMT on this repo once we wrap up the discussion around jdk8.